pandas/tools/tests/test_util.py

```diff
@@ -93,6 +93,11 @@ def test_set_locale(self):
             raise nose.SkipTest("Only a single locale found, no point in "
                                 "trying to test setting another locale")
 
+        if all(x is None for x in CURRENT_LOCALE):
+            # Not sure why, but on some travis runs with pytest,
```

Comment:
ok, though very odd that this occurs only in pytest......let's put a note to try to remove this later